addon/components/model-form.hbs

```diff
@@ -40,6 +40,7 @@
       @onStatusCorrection={{@onStatusCorrection}}
       @getChangeParameters={{@getChangeParameters}}
       @getCorrectionParameters={{@getCorrectionParameters}}
+      @getChangeHint={{@getChangeHint}}
```

Comment:
This component has definitely grown too big by now :sweat_smile: if we have time in the future we should check if we can split it up somehow.

Reply:
Yup, it hurt a bit adding yet another field :sweat_smile: